keps/sig-scheduling/20190311-resource_bin_packing_priority_function.md

```diff
@@ -151,11 +151,18 @@ The Scheduler should submit the 2 resource job on Node 3 as the utilization is h
 
 ### Test Plan
 
-_To be filled until targeted at a release._
+- **Unit Tests:** All changes must be covered by unit tests.
+- **Integration Tests / E2E Tests:** The use cases discussed in this KEP must
```

Comment:
I don't think e2e tests are needed for this feature. Unit and integration tests should be enough.